utils/rocvideodecode/roc_video_dec.cpp

```diff
@@ -247,6 +250,7 @@ static void GetSurfaceStrideInternal(rocDecVideoSurfaceFormat surface_format, ui
 *  0: fail, 1: succeeded, > 1: override dpb size of parser (set by CUVIDPARSERPARAMS::max_num_decode_surfaces while creating parser)
 */
 int RocVideoDecoder::HandleVideoSequence(RocdecVideoFormat *p_video_format) {
+    std::chrono::_V2::system_clock::time_point start_time = StartTimer();
```

Comment:
you can use `auto` instead of `std::chrono::_V2::system_clock::time_point` here.